nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -61,12 +60,15 @@ in
           difficulty = mkOption {
             type = types.enum [ "easy" "medium" "hard" ];
             default = "medium";
-            description = "The difficultly of the captcha to solve.";
+            description = "The difficulty of the captcha to solve.";
           };
         };
 
-        options.database.createLocally = mkEnableOption "creation of database on the instance";
-
+        options.database.createLocally = mkOption {
+          type = types.bool;
+          default = true;
+          description = "Whether to create a local database automatically.";
```

Comment:
this needs to be rephrased. Right now it suggests we are using sqlite and don't configure the nixos postgres service.